portal-web/docroot/html/taglib/ui/page_iterator/start.jsp

```diff
@@ -83,13 +83,14 @@ NumberFormat numberFormat = NumberFormat.getNumberInstance(locale);
 
 		<c:if test="<%= deltaConfigurable %>">
 			<div class="dropdown pagination-items-per-page">
-				<button aria-describedby="<%= ariaPaginationResults %>" aria-haspopup="true" class="dropdown-toggle page-link" data-toggle="liferay-dropdown">
+				
+				<button aria-labelledby="<%= ariaPaginationResults %>" aria-expanded="false" aria-controls="pagination-picker" aria-haspopup="listbox" class="dropdown-toggle page-link" data-attribute="<%= delta %>" data-toggle="liferay-dropdown" role="combobox">
```

Comment:
Why are we switching from `aria-describedby` to `aria-labelledby`? The clay pagination component uses aria-described by for this: https://clayui.com/docs/components/pagination-bar.html

Reply:
I was following [Picker component](https://clayui.com/docs/components/picker.html) markup, as requested by @marcoscv-work. But make sense to leave as `aria-describedby` since we are using a bigger description.

Reply:
Yep, the pagination is also using the Picker component, so this is a better example to work from since it's contextualized to the exact problem you're trying to solve.